yarn-project/aztec-rpc/src/aztec_rpc_server/aztec_rpc_server.test.ts

```diff
@@ -19,7 +20,10 @@ describe('AztecRpcServer', function () {
     keyStore = new TestKeyStore(await Grumpkin.new());
     node = mock<AztecNode>();
     db = new MemoryDB();
-    rpcServer = new AztecRPCServer(keyStore, node, db);
+    const config: RpcServerConfig = {
+      l2BlockPollingIntervalMS: 1000,
```

Comment:
shouldn't we set this lower for tests to run faster?

Reply:
Done